recipients_app/lib/view/widgets/survey/survey_card_container.dart

```diff
@@ -1,20 +1,22 @@
-import "package:app/data/models/survey/survey_card_status.dart";
-import "package:app/data/models/survey/survey_ui_state.dart";
+import "package:app/core/cubits/survey/survey_cubit.dart";
+import "package:app/data/models/survey/mapped_survey.dart";
 import "package:app/ui/configs/configs.dart";
+import "package:app/view/widgets/dashboard_item.dart";
 import "package:app/view/widgets/survey/survey_card_bottom_action.dart";
 import "package:flutter/material.dart";
 
-class SurveyCard extends StatelessWidget {
-  const SurveyCard({super.key});
+class SurveyCardContainer extends DashboardItem {
+  final MappedSurvey mappedSurvey;
+  final SurveyCubit surveyCubit;
+
+  const SurveyCardContainer({
+    super.key,
+    required this.mappedSurvey,
+    required this.surveyCubit,
+  });
 
   @override
   Widget build(BuildContext context) {
-    final SurveyUiState surveyUiState = const SurveyUiState(
-      status: SurveyCardStatus.firstReminder,
-      daysToDeadline: 10,
-      answeredDate: null,
-    );
-
     return Card(
```

Comment:
Padding can be const

Reply:
Column hasn't got the const constructor as far as I can see.

Reply:
its for the new flutter version, missed that. we will update in the next step